wbooks/src/redux/authorization/actions.ts

```diff
@@ -14,7 +13,7 @@ export const actions = {
 export const actionCreators = {
   login: (credentials: Credentials) => async (dispatch: Dispatch) => {
     dispatch({ type: actions.LOGIN });
-    const response: ApiResponse<LoginResponse, string> = await login(credentials);
+    const response: LoginApiResponse = await login(credentials);
```

Comment:
You should be able to delete this part
```suggestion
    const response = await login(credentials);
```

Reply:
Yes, you're right